### PR TITLE
label machine when kubelet start

### PR DESCRIPTION
**Pull request by YitongFeng**
no need-label.sh script anymore

### CODE REVIEW

File: cluster-configuration/cluster-configuration.yaml [hunk 1]
```diff
@@ -104,7 +103,7 @@ machine-list:
       #username: username (Optional)
       #password: password (Optional)
       k8s-role: worker
-      pai-worker: "true"
```
Comment (ydye) [inline cluster-configuration/cluster-configuration.yaml]: ```pai-worker: ture``` will be better

File: cluster-configuration/cluster-configuration.yaml [hunk 2]
```diff
@@ -60,9 +60,8 @@ machine-list:
       k8s-role: master
       dashboard: "true"
       zkid: "1"
-      pai-master: "true"
-      watchdog: "true"
-      alert-manager: "true"
+      pai-role: master
```
Comment (ydye) [inline cluster-configuration/cluster-configuration.yaml]: ```pai-master: ture``` will be better

File: pai-management/k8sPaiLibrary/template/kubelet.sh.template [hunk 3]
```diff
@@ -46,6 +46,7 @@ docker run \
   --pod-manifest-path=/etc/kubernetes/manifests \
   --allow-privileged=true \
   --logtostderr=true \
+  --node-labels pai-{{ hostcofig['pai-role'] }}=true \
```
Comment (YitongFeng) [inline pai-management/k8sPaiLibrary/template/kubelet.sh.template]: This line will changed to "pai-master=true" label. @ydye

File: pai-management/k8sPaiLibrary/template/kubelet.sh.template [hunk 4]
```diff
@@ -46,6 +46,8 @@ docker run \
   --pod-manifest-path=/etc/kubernetes/manifests \
   --allow-privileged=true \
   --logtostderr=true \
+  --node-labels {% if 'pai-master' in hostcofig %} pai-master=true {% endif %} \
+  --node-labels {% if 'pai-worker' in hostcofig %} pai-worker=true {% endif %} \
   --pod-infra-container-image {{ clusterconfig['dockerregistry'] }}/pause-amd64:3.0 \
```
[the remark this answers is not in the document]
Comment (ydye) [reply pai-management/k8sPaiLibrary/template/kubelet.sh.template]:
```
{% if 'pai-worker' in hostcofig -%} 
  --node-labels  pai-worker=true \
{%- endif %}
```

**Comment by YitongFeng** [comment]
Combine into PR #1081 , split these two PR will breakdown Jenkins pipeline

**Pull request closed without merging**